cobra/io/dict.py

```diff
@@ -147,7 +152,10 @@ def reaction_from_dict(reaction, model):
                 (model.metabolites.get_by_id(str(met)), coeff)
                 for met, coeff in iteritems(v)))
         else:
-            setattr(new_reaction, k, v)
+            if k == "lower_bound" or k == "upper_bound":
+                setattr(new_reaction, k, float(v))
+            else:
+                setattr(new_reaction, k, v)
```

Comment:
It is not clear why a loop is needed to cover 6 cases half of which receive a special treatment within the loop (`metabolites`, `lower_bound`, and `upper_bound`). Why not also treating the remaining 3 cases directly?

Reply:
Regarding the use of `for` loop, again, I would request COBRApy community to provide their review for it.

Reply:
Loop looks fine. Is standard practise here. There could be additional attributes in the future.